controllers/auth.py

```diff
@@ -1,5 +1,83 @@
-def controller_endpoints(app):
-    @app.route('/', methods=['GET'])
-    def home():
-        print("Entrando")
-        return "Hola"
+from flask import request, jsonify
+import re
+from services.user import create_user
+
+import services.user as user_service
+
+from services.auth import auth_user
+import hashlib
+from datetime import datetime, timezone
+import secrets
+
+
+def controller(app):
+    @app.route('/', methods=["GET"])
+    def start():
```

Comment:
what's te purpose of this `start` function?

Reply:
Only testif the service is online but, i need to delete! Is irrelevant for the api functionallity